internal/flink/command_artifact.go

```diff
@@ -13,11 +13,12 @@ type flinkArtifactOut struct {
 	Id            string `human:"ID" serialized:"id"`
 	Name          string `human:"Name" serialized:"name"`
 	Version       string `human:"Version" serialized:"version"`
-	Class         string `human:"Class" serialized:"class"`
 	Cloud         string `human:"Cloud" serialized:"cloud"`
 	Region        string `human:"Region" serialized:"region"`
 	Environment   string `human:"Environment" serialized:"environment"`
 	ContentFormat string `human:"Content Format" serialized:"content_format"`
+	Description   string `human:"Description" serialized:"description"`
+	DocLink       string `human:"Documentation link" serialized:"doc_link"`
```

Comment:
We should make it consistent between the field name, the human tag, and the serialized tag.

And we prefer not to abbreviate single words, so maybe `DocumentationLink`, `Documentation Link` and `documentation_link`